cluster-operator/src/main/java/io/strimzi/operator/cluster/model/AbstractModel.java

```diff
@@ -754,6 +756,19 @@ protected Service createHeadlessService(List<ServicePort> ports, Map<String, Str
         return service;
     }
 
+    /**
+     * @return if the storage is a persistent claim one or a jbod containing at least a persistent volume
+     */
+    private boolean isPersistentClaimStorage() {
```

Comment:
Method name is a little confusing: We have `PersistentClaimStorage`, but the method's not simply checking for that case. Something like `isStoragePersistent()` or `containsPersistentStorage()` would be clearer. I also think this could be a static method of `ModelUtils`, taking a `storage` parameter.

Reply:
I guess that it's in the plan of refactoring/removing the `AbstractModel` class. We should talk more about that because I don't know if having a growing `ModelUtils` class with a bunch of methods inside is the right way. Btw I am going to move the method there for now.